src/Shared/Tests/Parallel.runsettings

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Are we configuring something to use this file?

Reply:
I'd like to have it used by default when running tests but my understanding is that it requires a reference in the command line.  Suggestions welcome.

Reply:
I think MSBuild properties are you best bet. https://learn.microsoft.com/en-us/visualstudio/test/configure-unit-tests-by-using-a-dot-runsettings-file?view=vs-2022#set-a-build-property